server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -159,6 +161,43 @@ func (r *Runner) Plan(ctx workflow.Context, root *terraform.LocalRoot, serverURL
 	return response, nil
 }
 
+func (r *Runner) PolicyCheck(ctx workflow.Context, root *terraform.LocalRoot, serverURL *url.URL, showFile string) error {
+	jobID, err := sideeffect.GenerateUUID(ctx)
+	if err != nil {
+		return errors.Wrap(err, "generating job id")
+	}
+
+	// fail if we error here since all successive calls to update this will fail otherwise
+	if err := r.Store.InitPolicyCheckJob(jobID, serverURL); err != nil {
+		return errors.Wrap(err, "initializing job")
+	}
```

Comment:
Does this imply we're going to stream logs?

Reply:
I'm not against it, was just curious. Also just want to understand if you consider this a success criteria for the project because it is technically a new feature.  I do think this is going to be great though because it would remove so much noise.

Reply:
Yep consolidating the plan and policy check runs is another goal of the project